website/docs/r/kubernetes_cluster_node_pool.html.markdown

```diff
@@ -137,6 +137,8 @@ The following arguments are supported:
 
 ~> At this time there's a bug in the AKS API where Tags for a Node Pool are not stored in the correct case - you [may wish to use Terraform's `ignore_changes` functionality to ignore changes to the casing](https://www.terraform.io/docs/configuration/resources.html#ignore_changes) until this is fixed in the AKS API.
 
+* `scale_down_mode` - (Optional) Used to specify the cluster autoscaler behavior. Allowed values are `Delete` and `Deallocate`. Defaults to `Delete`.
+
```

Comment:
This feature has some limitations (not applicable for spot node pools) and is in preview. So an additional note here would be useful 
```suggestion
* `scale_down_mode` - (Optional) Used to specify the cluster autoscaler behavior. Allowed values are `Delete` and `Deallocate`. Defaults to `Delete`.

-> **Note:** Cannot be set when `priority` is set to Spot. 
~> **Note:** Scale down mode is in Public Preview - more information and details on how to opt into the preview can be found in [this article](https://docs.microsoft.com/en-us/azure/aks/scale-down-mode)
```
```